File: Python/question4.py
def totalFibonacci(n):
   def fibonacciRecursive(num):
       if (num <= 1):
           return 1
       else:
           return fibonacciRecursive(num - 1) + fibonacciRecursive(num - 2)
   return fibonacciRecursive(n) + n
print(totalFibonacci(3))

File: Javascript/question4.js
function totalFibonacci(n){
    function fibonacciRecursive(num) {
        if (num <= 1) return 1;
   
        return fibonacciRecursive(num - 1) + fibonacciRecursive(num - 2);
    }
    return fibonacciRecursive(n) + n;
 }
 
 
module.exports.totalFibonacci = totalFibonacci;